server/httpHandler.go

```diff
@@ -37,6 +40,26 @@ var postActionIntegrationRequestFromJson = model.PostActionIntegrationRequestFro
 var notifyHandlerError = defaultNotifyHandlerError
 
 func (p *Plugin) handleHTTPRequest(w http.ResponseWriter, r *http.Request) {
+	if r.URL.Path == "/giphyAttribution" {
+		bundlePath, err := p.API.GetBundlePath()
+		if err != nil {
+			p.API.LogError("error getting bundle path")
+			return
+		}
+
+		buffer, err := ioutil.ReadFile(filepath.Join(bundlePath, "assets", "Poweredby_100px-White_VertLogo.png"))
```

Comment:
How about renaming the file to make it generic, so that changing it, for example to Poweredby_100px-Black_VerLogo.png, won't require to recompile the go package?

Reply:
Yep, great suggestion and done.